devtools/gossipwith.c

```diff
@@ -21,6 +21,7 @@
 #include <stdio.h>
 #include <wire/peer_wire.h>
 
+#define chainparams_get_ln_port simple_get_ln_port
```

Comment:
You shouldn't need to hack this one: the tool sets chainparams correctly, and so will Just Work, whereas with this change it will always try to connect in 9735, even if they specify --network...